src/rulesets/tests/__snapshots__/lifecycle.test.ts.snap

```diff
@@ -54,7 +54,7 @@ Object {
       },
       "condition": "not change unless it was wip",
       "docsLink": "https://github.com/snyk/sweater-comb/blob/main/docs/version.md#promoting-stability-of-a-resource-over-time",
-      "error": "expected 'ga' to equal 'beta'",
+      "error": "stability transition from 'ga' to 'beta' not allowed",
```

Comment:
this one actually says what's wrong. A user might read the first and switch back to ga, but will they learn?